src/App.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/extensions
+import Calculator from './components/Calculator.js';
```

Comment:
- [ ] [REQUIRED] I got a server error while trying to run the application.
<img width="1285" alt="Screenshot 2023-03-22 at 16 51 18" src="https://user-images.githubusercontent.com/58692590/226966441-4e12ffd8-94d5-49e4-b383-e0e9a35c64b3.png">

This error message suggests that there is a discrepancy between the import statement for the Calculator component in your App.js file and the actual filename of the Calculator.js file in the src/components directory.

The import statement in App.js is looking for a file named Calculator.js, but the actual filename is calculator.js (with a lowercase "c"). This mismatch in the filename casing is causing the module not found error.

The appropriate naming convention for Components states that they should start with an Uppercase, therefor your component name should be:

```
Calculator.js
```
Let's fix this @maherramadan78 👍🏽